editor/editor_notes.cpp

```diff
@@ -148,8 +150,16 @@ void Notes::CreateNote(ms::LatLon const & latLon, string const & text)
     return;
   }
 
-  lock_guard<mutex> g(m_mu);
-  m_notes.emplace_back(latLon, text);
+  std::lock_guard<std::mutex> g(m_mu);
+  auto const it = std::find_if(m_notes.begin(), m_notes.end(), [&latLon](Note const & note) {
+    return my::AlmostEqualAbs(latLon.lat, note.m_point.lat, 1e-7) &&
```

Comment:
may create such helper somewhere in base - this code duplicated in this PR - here and in tests.